Felienne/hedy
# [FEATURE] Implements dynamic (printable) cheatsheets

**Description**
With #2193 @MarleenGilsing made some great cheatsheet that can be used as reference material for kids. However, as they are .pdf files we are unable to dynamically change them dependent on language and keyword language. In this PR we implement a dynamically created cheatsheet structure, automatically generating them based on the language and keyword language. Currently we use the `level-defaults` to create the files but as they differ from the current cheatsheet and don't give that much extra information it might be useful to create some dedicated cheatsheet YAMLs.

**Fixes**
This PR fixes #2198

**How to test**
Navigate to `/cheatsheet/<level>` and verify that the correct level-defaults are shown. Also verify that the language and keyword language are correct based on the user preferences.

![Schermafbeelding 2022-03-14 om 13 09 17](https://user-images.githubusercontent.com/38155424/158169576-ecba817a-546e-48fd-8b39-931db527a9a2.png)
